scripts/environment/prepare.sh

```diff
@@ -15,7 +15,8 @@ if [[ "$(cargo-nextest --version)" != "cargo-nextest 0.9.72" ]] ; then
   rustup run stable cargo install cargo-nextest --version 0.9.72 --force --locked
 fi
 if ! cargo deny --version >& /dev/null ; then
-  rustup run stable cargo install cargo-deny --force --locked
```

Comment:
This should have been installing latest so I'm actually a little confused why it wasn't flagging the unmaintained warnings as errors. In any event, I think it is better to lock it.

This could be replaced with aqua in the future.